runtime/vm/segment.c

```diff
@@ -471,7 +482,15 @@ static J9MemorySegment * allocateVirtualMemorySegmentInListInternal(J9JavaVM *ja
 	if(segment == NULL) {
 		Trc_VM_allocateMemorySegmentInList_EntryAllocFailed(segmentList, type);
 	} else {
+		PORT_ACCESS_FROM_JAVAVM(javaVM);
+		UDATA pageSize = j9vmem_supported_page_sizes()[0];
+
 		segment->type = type;
+
+		if (J9_ARE_ALL_BITS_SET(type, MEMORY_TYPE_RAM_CLASS | MEMORY_TYPE_DISCLAIMABLE_TO_FILE)) {
+			size += pageSize;
```

Comment:
Could you please add a comment as to why we increase the size of the segment by one page?